app/src/pages/header/components/layouts/HeaderComponent.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { GlobalWidthComponent } from '../../../components/GlobalWidthComponent';
```

Comment:
You can bind the alias to components path:
```
webpack:
resolve: {
        extensions: ['.js', '.jsx'],
        alias: {
            '@': util.src,
            '@models': `${util.src}/models`,
            '@components': `${util.src}/components`,
        }
    },
    
component:    
import { Hello, About, Home, Users } from '@components'    
```

Reply:
Nice idea, thanks) Fixed